chromium_src/chrome/browser/permissions/chrome_permissions_client.cc

```diff
@@ -28,18 +28,14 @@ bool ChromePermissionsClient::BraveCanBypassEmbeddingOriginCheck(
     ContentSettingsType type) {
   // Since requesting_origin has been overwritten by us to add address info,
   // it will fail Chromium's origin check because requesting_origin is now
-  // different from the embedding_origin. To address this, we get the original
-  // requesting origin back and use it to check with embedding_origin instead,
-  // and let it bypass the origin check from Chromium when the original
-  // requesting_origin & embedding_origin are the same.
+  // different from the embedding_origin.
   url::Origin original_requesting_origin;
   if ((type == ContentSettingsType::BRAVE_ETHEREUM ||
        type == ContentSettingsType::BRAVE_SOLANA) &&
       brave_wallet::ParseRequestingOriginFromSubRequest(
           permissions::ContentSettingsTypeToRequestType(type),
           url::Origin::Create(requesting_origin), &original_requesting_origin,
-          nullptr) &&
-      original_requesting_origin == url::Origin::Create(embedding_origin)) {
+          nullptr)) {
```

Comment:
Please remove the obsolete comment, we should instead document here why BRAVE_ETHEREUM and BRAVE_SOLANA should be excluded from the origin checks.
And we don't need to run ParseRequestingOriginFromSubRequest anymore,  just type checking.

Reply:
Fixed in b4c5b78a40ec74bc5f9b1d9bbb1e54597e2c2cf1